app/src/i18n/locales/en/auth.json

```diff
@@ -54,5 +54,14 @@
 
   "update-password-heading": "Update your Password",
   "update-password-details": "Update your password. This is the one that you will use from now on.",
-  "reset-button": "Reset Password"
+  "reset-button": "Reset Password",
+
+  "password-min-length": "Password must be at least 8 characters long.",
```

Comment:
Let's remind the product team that we need translations here. Good to merge without them though.

Reply:
Awesome